config/initializers/_config.rb

```diff
@@ -52,5 +52,6 @@ module Convection
     vibrations_url:
       ENV['VIBRATIONS_URL'] || 'https://admin-partners-staging.artsy.net',
     unleash_url: ENV['UNLEASH_URL'],
-    unleash_token: ENV['UNLEASH_TOKEN']
+    unleash_token: ENV['UNLEASH_TOKEN'],
+    send_new_receipt_email: ENV['SEND_NEW_RECEIPT_EMAIL'] == 'true'
```

Comment:
=== ? 🤔

Reply:
`==` suffice in Ruby. [here](https://stackoverflow.com/questions/7156955/whats-the-difference-between-equal-eql-and) is some explanation if you are curious.